doc/guides/tips-tricks-and-troubleshooting.md

```diff
@@ -0,0 +1,74 @@
+# Tips, Tricks & Troubleshooting
```

Comment:
Maybe `Tips and Troubleshooting for Fhqwhgads` where `Fhqwhgads` is some specific activity? Like, is it "Tips for Proposing Pull Requests"? Maybe it's more broad than that, but some kind of thing that at least provides context for someone coming from Google that this isn't about troubleshooting running Node.js in production or something like that.